README.md

```diff
@@ -1,3 +1,484 @@
+  - [gst-concurrent-videoplay-composition](#qimsdk-lmp)
+  - [gst-ai-classification](#gst-ai-classification)
+  - [gst-ai-daisychain-detection-classification](#gst-ai-daisychain-detection-classification)
+  - [gst-ai-monodepth](#gst-ai-monodepth)
+  - [gst-ai-object-detection](#gst-ai-object-detection)
+  - [gst-ai-parallel-inference](#gst-ai-parallel-inference)
+  - [gst-ai-pose-detection](#gst-ai-pose-detection)
+  - [gst-ai-segmentation](#gst-ai-segmentation)
+  - [gst-multi-camera-example](#gst-multi-camera-example)
+  - [Deploy](#deploy)
+- [Fioctl](#fioctl)
+  - [View Targets](#view-targets)
+  - [View Devices](#view-devices)
+  - [Change Device Tag](#view-devices)
+  - [Change Device Application](#change-device-application)
+  - [Deploy shellhttpd Application](#deploy-shellhttpd-application)
+  - [Deploy gst-concurrent-videoplay-composition Application](#deploy-gst-concurrent-videoplay-composition-application)
+  - [Deploy gst-ai-classification Application](#deploy-gst-ai-classification-application)
+  - [Deploy gst-ai-daisychain-detection-classification Application](#deploy-gst-ai-daisychain-detection-classification-application)
+  - [Deploy gst-ai-monodepth Application](#deploy-gst-ai-monodepth-application)
+  - [Deploy gst-ai-object-detection Application](#deploy-gst-ai-object-detection-application)
+  - [Deploy gst-ai-parallel-inference Application](#deploy-gst-ai-parallel-inference-application)
+  - [Deploy gst-ai-pose-detection Application](#deploy-gst-ai-pose-detection-application)
+  - [Deploy gst-ai-segmentation Application](#deploy-gst-ai-segmentation-application)
+  - [Deploy gst-multi-camera-example Application](#deploy-gst-multi-camera-example-application)
+  - [Enable/Disable Wireguard VPN](#enabledisable-wireguard-vpn)
+- [FoundriesFactory Images](#foundriesfactory-images)
+  - [LmP Factory Image](#lmp-factory-image)
+- [Wireguard VPN](#wireguard-vpn)
+- [Updating FoundriesFactory Images](#updating-foundriesfactory-images)
+- [Useful Links](#updating-foundriesfactory-images)
+
+## Document Revision History
+
+| Version | Date       | Comments        |
+|---------|------------|-----------------|
+| v0.1    | 2024-07-01 | Initial release |
+
+## Definitions, Acronyms, and Abbreviations
+
+| Variable        | Meaning                                                           |
+|-----------------|-------------------------------------------------------------------|
+| `<factory>`     | Your factory name                                                 |
+| `<device_name>` | Device name used when you registered the device                   |
+| `<app_name>`    | Docker-compose app available to run on the device                 |
+| `<tag_name>`    | The branch you want your device to follow                         |
+| `<IP>`          | Device IP                                                         |
+
+## Getting Started
+
+To get started using the FoundriesFactory and the Linux microPlatform, please follow the steps below:
```

Comment:
```suggestion
To get started using the FoundriesFactory™ Platform and the Linux microPlatform, please follow the steps below:
```
If we don't want to use "Platform" as it is followed by "microPlatform", the alternatives, in preferred order are:
* Sass
* Service
* Software
* Solution